modules/attachments/variables.tf

```diff
@@ -0,0 +1,49 @@
+variable "profile_id" {
+  type        = string
+  description = "The profile ID"
+}
+
+variable "scc_instance_id" {
+  type        = string
+  description = " The ID of the SCC instance in a particular region"
+}
+
+variable "attachment_name" {
+  type        = string
+  description = "The name of the SCC profile attachment"
+}
+
+variable "attachment_description" {
+  type        = string
+  description = "The description for the SCC profile attachment"
+}
+
+variable "attachment_schedule" {
+  type        = string
+  description = "The schedule of an attachment evaluation. Allowable values are: daily, every_7_days, every_30_days"
+
+  validation {
+    condition     = contains(["daily", "every_7_days", "every_30_days"], var.attachment_schedule)
+    error_message = "Allowed schedule can be - daily, every_7_days, every_30_days."
+  }
+}
+
+variable "attachment_status" {
```

Comment:
I'm a little confused as to what status means here. Maybe this is the how you disable any schedule for the attachment. I have reached out to SCC team to confirm. If this is the case, we may make this easier in our module and just update the `attachment_schedule` to allow users to pass the string "None" and then in our code, we could have logic to then set status to disabled if thats the case.

Reply:
It would then match what the UI shows:
![image](https://github.com/terraform-ibm-modules/terraform-ibm-scc/assets/29608224/5e52717a-f497-4532-aaab-08b69b71cd79)

Reply:
@iamar7 SCC team confirmed that setting status to disabled here is the same thing as setting the schedule to None. So can we remove the `attachment_status` variable and just update `attachment_schedule` variable to allow "None" to be passed to it. In our tf logic we can then set the appropriate `status` value to the provider resource block. Make sense?

Reply:
@iamar7 this has not been addressed yet